express/code/blocks/frictionless-quick-action-mobile/frictionless-quick-action-mobile.css

```diff
@@ -43,6 +43,7 @@
     width: 100%;
     margin: auto;
     transition: opacity 0.2s;
+    height: 667px;
```

Comment:
is this value a site wide expected size we expect to use somewhere?

Reply:
it's a hardcoded value from the beginning and I'm resolving a duplicate style so not introducing it... but a good callout and we should definitely clean up things like this... I can undo this change too to make my intended changes more clear

Reply:
all good! thanks for the background insight. I'd say, lets bring it to a variable to make it more visible and easier to track.

Reply:
updated to a var! Thanks!

Reply:
🙇‍♂️